ReadMe.rst

```diff
@@ -0,0 +1,91 @@
+Project Mu UEFI Variables
+=========================
+
+This repository contains a prototype for a new UEFI variable design for modern PCs.
+
+It is extensible, secure, with built-in testing up and down the entire stack. The design and implementation are all
+taking place here in this repo.
+
+Project Mu Top-Level information
+________________________________
+
+This repository is part of Project Mu. Please see Project Mu for details: https://microsoft.github.io/mu.
+
+Contributing
+============
+
+We welcome everyone to file feature requests, bugs, participate in code reviews, submit code, update documentation,
+and help us build the best variable driver possible.
+
+At this time, we are very early in the work so we're particularly interested in ideas around the future of UEFI
+variables (including breaking UEFI specification compatibility) and suggestions to help shape the overall design.
+
+Please open GitHub issues directly in this repo.

```

Comment:
@spbrogan, I know most issues are collected in the mu repo. What do you think about trying to use milestones and track feature specific work directly in this repo?

Reply:
sounds good.